Modules/User/Assets/js/components/AsgardPermissions.vue

```diff
@@ -46,7 +46,7 @@
                                 </div>
                             </div>
                             <div class="col-md-9">
-                                <el-radio-group v-model="permissions[`${subPermissionTitle}`][`${permissionAction}`]">
+                                <el-radio-group v-model="permissions[subPermissionTitle][permissionAction]">
```

Comment:
Isn't this the same? :D

Reply:
It is not. `v-model` doesn't accept a string, but a reference to a variable.

The way I did the string was wrong anyway, if it did accept a string. It should have been `` `permissions[${subPermissionTitle}][${subPermissionTitle}]` ``